documentation/OtherMLPackages.md

```diff
@@ -0,0 +1,62 @@
+# Testing Other ML Packages
+
+In our documentation, we state that the mitigation algorithms in Fairlearn are compatible with any package which presents an interface similar to SciKit-Learn.
+Specifically, so long as there's some sort of 'estimator' concept which provides `fit()` and `predict()` methods (the former with a `sample_weight=` argument), then those estimators can be used with our mitigation algorithms.
+However, we do not currently have tests proving that we do offer this sort of compatibility with a variety of popular ML packages.
+
+## Target Packages
+
+We should show compatibility with the following ML packages:
+  - XGBoost
+  - LightGBM
+  - TensorFlow
```

Comment:
The discussion in https://github.com/fairlearn/fairlearn/issues/439 may be helpful for articulating the value that users would get in using fairlearn's mitigation techniques over TFCO's.

Reply:
Certainly, although that's not the immediate point of this (it will help towards that goal if we want to run direct comparisons between TensorFlow and Fairlearn mitigations). This proposal is just about 'can we run an estimator from X package in the Fairlearn mitigation algorithms?'